packages/flex-dev-utils/src/packages/lib/__tests__/getLatestFlexUIVersion.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as getRegistryVersionScripts from '../getRegistryVersion';
+import getLatestFlexUIVersion from '../getLatestFlexUIVersion';
+
+describe('getLatestFlexUIVersion', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  it('should return the latest verison', async () => {
+    const getRegistryVersion = jest.spyOn(getRegistryVersionScripts, 'default');
+    // @ts-ignore
```

Comment:
What's this `@ts-ignore` for? We rarely should be using ts-ignore. If you are using it multiple times throughout the code, then you probably should create a wrapper function for this